packages/@aws-cdk/aws-synthetics/lib/runtime.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * All known Lambda runtime families.
+ */
+export enum RuntimeFamily {
```

Comment:
let's also make this a class with static properties. as it stands, customers could not unblock themselves to add a new runtime if the runtime used a family we did not support